classes/Task/Backup/BackupFiles.php

```diff
@@ -46,7 +46,7 @@ class BackupFiles extends AbstractTask
     public function run(): int
     {
         $this->stepDone = false;
-        $backupFilesFilename = $this->container->getState()->getBackupFilesFilename();
+        $backupFilesFilename = $this->container->getBackupState()->getBackupFilesFilename();
```

Comment:
I think you can add a variable to not reuse `$this->container->getBackupState()` all the time but just `$backupState` or `$state`